frontend/src/components/teacher/dashboard/AssessmentsSection.tsx

```diff
@@ -56,34 +58,56 @@ const AssessmentsSection = () => {
         </Box>
       )}
       {!!data?.length && !error && !loading && (
-        <Tabs onChange={(index) => setCurrentTab(TEST_SESSION_STATUSES[index])}>
-          <TabList border="none" gap={8}>
-            {TEST_SESSION_STATUSES.map((status) => (
-              <Tab
-                key={status}
-                _focus={{ background: "none" }}
-                border="none"
-                color="blue.100"
-                fontWeight="bold"
-                px={0}
-              >
-                {titleCase(status)}
-              </Tab>
-            ))}
-          </TabList>
-          <TabPanels>
-            {TEST_SESSION_STATUSES.map((status) => (
-              <TabPanel key={status} p={0}>
-                {sortedData?.map((session) => (
-                  <TestSessionListItem
-                    key={session.testSessionId}
-                    session={session}
-                  />
-                ))}
-              </TabPanel>
-            ))}
-          </TabPanels>
-        </Tabs>
+        <>
+          <Tabs
+            onChange={(index) => setCurrentTab(TEST_SESSION_STATUSES[index])}
+          >
+            <TabList border="none" gap={8}>
+              {TEST_SESSION_STATUSES.map((status) => (
+                <Tab
+                  key={status}
+                  _focus={{ background: "none" }}
+                  border="none"
+                  color="blue.100"
+                  fontWeight="bold"
+                  px={0}
+                >
+                  {titleCase(status)}
+                </Tab>
+              ))}
+            </TabList>
+            <TabPanels>
+              {TEST_SESSION_STATUSES.map((status) => (
+                <TabPanel key={status} p={0}>
+                  {sortedData?.map((session) => (
+                    <TestSessionListItem
+                      key={session.testSessionId}
+                      session={session}
+                    />
+                  ))}
+                </TabPanel>
+              ))}
+            </TabPanels>
+          </Tabs>
+          <RouterLink
```

Comment:
Instead of having all this extra styling, is there any chance we can extract a parent component from the list item component and this? Might make it easier to make visually consistent changes in the future